{"name": "test_create_failure_reason", "status": "passed", "description": "Should able to create failure reason", "start": 1678887707307, "stop": 1678887708549, "uuid": "3aab1a3a-4f80-4635-b1e5-760131b81aff", "historyId": "60ea155cbc409e52f1d6514719c0e268", "testCaseId": "52546f71e61a22bbd443f829bfcacd4d", "fullName": "Test_Suites.test_all_module.Test_Failure_Reason#test_create_failure_reason", "labels": [{"name": "severity", "value": "CRITICAL"}, {"name": "tag", "value": "order(704)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "Test_Suites"}, {"name": "suite", "value": "test_all_module"}, {"name": "subSuite", "value": "Test_Failure_Reason"}, {"name": "host", "value": "sujith-HP-Compaq-Elite-8300-SFF"}, {"name": "thread", "value": "79826-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_Suites.test_all_module"}]}